projects/core/src/lib/components/edit/state/edit.reducer.ts

```diff
@@ -57,6 +66,67 @@ const onLoadEditFeaturesSuccess = (
   };
 };
 
+const onLoadCopyFeaturesSuccess = (
+  state: EditState,
+  payload: ReturnType<typeof EditActions.loadCopyFeaturesSuccess>,
+): EditState => {
+  state = {
+    ...state,
+    loadStatus: LoadingStateEnum.LOADED,
+  };
+  const geometry = payload.featureInfo[0]?.features[0]?.geometry;
+  if (!geometry) {
+    return state;
+  }
+
+  // Deselect a feature by checking if the geometry WKT is already in the copiedFeatures array (can't search by fid)
```

Comment:
Why not by fid? seems more reliable although clicking things on exactly the same position will probably mean you want to deselect that point...